content/contributing/contribute-code.md

```diff
@@ -4,11 +4,11 @@ The software that powers the XRP Ledger is open-source, so anyone can download,
 
 | XRP Ledger Source Code |                                                     |
 |:-----------------------|:----------------------------------------------------|
-| Repository             | <https://github.com/ripple/rippled/>                |
+| Repository             | <[https://github.com/XRPLF/rippled](https://github.com/XRPLF/rippled)>                |
```

Comment:
This was already an [autolink](https://daringfireball.net/projects/markdown/syntax#autolink), so there's no need to mix link styles (and doing both like this results in extra symbols)

```suggestion
| Repository             | <https://github.com/XRPLF/rippled>                |
```